GOVERNANCE.md

```
@@ -0,0 +1,207 @@
+
+# OpenCue Project Governance
+
+OpenCue is a project of the Academy Software Foundation and relies on the
+ASWF governance policies, supported by the Linux Foundation.
+
+There are three primary project roles: Contributors submit code to the
+project; Committers approve code to be included into the project; and the
+Technical Steering Committee (TSC) provides overall high-level project
+guidance.
+
+* [Contributors](#Contributors)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#contributors`; Actual: `#Contributors`

(MD051, link-fragments)